Rachnera Message Bust.rb

```diff
@@ -417,8 +417,8 @@ def bust_extra_x
   alias original_591_draw_face draw_face
   def draw_face(face_name, face_index, x, y, enabled = true)
     return if show_bust?
-    return original_591_draw_face(face_name, face_index, x + 2*standard_padding, y, enabled) if valid_context?
-    original_591_draw_face(face_name, face_index, x, y, enabled)
+    return original_591_draw_face(bust_face_name, face_index, x + 2*standard_padding, y, enabled) if valid_context?
+    original_591_draw_face(bust_face_name, face_index, x, y, enabled)
```

Comment:
This feature was way past the point where it was just going to be removed. But hacking the default behavior regardless of the switch officializes it in a way.

Reply:
Yeah, this is intentional since Sierra is keen on the ability to auto-swap appearances as a means of showing character progression, so I think it makes sense to apply the faceset swaps even if the dialogue busts are hidden. I guess it depends on whether you regard the setting as toggling the display of dialogue busts or disabling the script altogether.